frontend-plugin-core/src/main/java/com/github/eirslett/maven/plugins/frontend/lib/NodeExecutorConfig.java

```diff
@@ -3,43 +3,48 @@
 import java.io.File;
 
 public interface NodeExecutorConfig {
-  File getNodePath();
-  File getNpmPath();
-  File getWorkingDirectory();
-  Platform getPlatform();
+    File getNodePath();
+
+    File getNpmPath();
+
+    File getWorkingDirectory();
+
+    Platform getPlatform();
 }
 
 final class InstallNodeExecutorConfig implements NodeExecutorConfig {
 
-  private static final String NODE_WINDOWS = "\\node\\node.exe";
-  private static final String NODE_DEFAULT = "/node/node";
-  private static final String NPM = "/node/node_modules/npm/bin/npm-cli.js";
+    private static final String NODE_WINDOWS =
+        NodeInstaller.INSTALL_PATH.replaceAll("/", "\\\\") + "\\node.exe";
+
+    private static final String NODE_DEFAULT = NodeInstaller.INSTALL_PATH + "/node";
 
-  private final InstallConfig installConfig;
+    private static final String NPM = NodeInstaller.INSTALL_PATH + "/node_modules/npm/bin/npm-cli.js";
 
-  public InstallNodeExecutorConfig(InstallConfig installConfig) {
-    this.installConfig = installConfig;
-  }
+    private final InstallConfig installConfig;
 
-  @Override
-  public File getNodePath() {
-    String nodeExecutable = getPlatform().isWindows() ? NODE_WINDOWS : NODE_DEFAULT;
-    return new File(installConfig.getInstallDirectory() + nodeExecutable);
-  }
+    public InstallNodeExecutorConfig(InstallConfig installConfig) {
+        this.installConfig = installConfig;
+    }
 
-  @Override
-  public File getNpmPath() {
-    return new File(installConfig.getInstallDirectory() + Utils.normalize(NPM));
-  }
+    @Override
+    public File getNodePath() {
+        String nodeExecutable = getPlatform().isWindows() ? NODE_WINDOWS : NODE_DEFAULT;
+        return new File(this.installConfig.getInstallDirectory() + nodeExecutable);
+    }
 
+    @Override
+    public File getNpmPath() {
+        return new File(this.installConfig.getInstallDirectory() + Utils.normalize(NPM));
+    }
 
-  @Override
-  public File getWorkingDirectory() {
-    return installConfig.getWorkingDirectory();
-  }
+    @Override
```

Comment:
Fixing the code style can be done in a separate PR (to keep the line change count down on this PR)